plugins/liquid/refined-types-plugin/src/test/kotlin/arrow/meta/plugins/liquid/LiquidDataflowTests.kt

```diff
@@ -338,6 +361,51 @@ class LiquidDataflowTests {
       withoutPlugin = { compiles }
     )
   }
+
+  @Test
+  @Disabled
+  fun `nullability, 1`() {
+    """
+      ${imports()}
+      fun nully1(x: Int?): Int? {
+        val y = x?.let { 1 }
+        return y.post({ it > 0 }) { "greater than 0" }
+      }
+      """(
+      // it seems that Kotlin's warning overrides ours
+      withPlugin = { failsWith { it.contains("`nully1` fails to satisfy the post-condition") } },
```

Comment:
I suppose this is the case related to the warnings that you mentioned in our call. I created https://app.clickup.com/t/1e0yfab so we can look into it after this PR

Reply:
Yes, this is one of them, the other one is `bad predicates`.